pkg/proxy/iptables/proxier.go

```diff
@@ -1442,19 +1428,6 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 
-	// Drop the packets in INVALID state, which would potentially cause
-	// unexpected connection reset if nf_conntrack_tcp_be_liberal is not set.
-	// Ref: https://github.com/kubernetes/kubernetes/issues/74839
-	// Ref: https://github.com/kubernetes/kubernetes/issues/117924
-	if !proxier.conntrackTCPLiberal {
```

Comment:
I only meant to drop this from nftables; we had previously agreed it would be a regression / "API break" to change the default behavior in iptables mode, but we have more flexibility with the nftables backend.